packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -96,6 +96,7 @@ class PouchLink extends CozyLink {
     this.storage = new PouchLocalStorage(
       options.platform?.storage || platformWeb.storage
     )
+    this.ignoreWarmup = ignoreWarmup
```

Comment:
If we don't specify any warmup query, this is useless, right?

Reply:
Done in #1524